sirepo/package_data/static/js/raydata.js

```diff
@@ -8,11 +8,11 @@ SIREPO.app.config(() => {
         <div data-ng-switch-when="DateTimePicker" data-ng-class="fieldClass">
           <div data-date-time-picker="" data-model="model" data-field="field"></div>
         </div>
-        <div data-ng-switch-when="QueuedScansTable" class="col-sm-12">
-          <div data-scans-table="" data-model-name="modelName"></div>
-        </div>
         <div data-ng-switch-when="CompletedScansTable" class="col-sm-12">
-          <div data-scans-table-with-modal="" data-model-name="modelName"></div>
+          <div data-scans-table="" data-model-name="modelName" data-analysis-status="executed"></div>
+        </div>
+        <div data-ng-switch-when="QueuedScansTable" class="col-sm-12">
+          <div data-scans-table="" data-model-name="modelName" data-analysis-status="queued"></div>
```

Comment:
The modal is on the queued tab as well, is that what we want?

Reply:
Yes for running scans it is nice to see the output they have produced so far.